http/api/tokenpki.go

```diff
@@ -55,14 +46,15 @@ func GetCertTokenPKIHandler(store TokenPKIStorer, logger log.Logger) http.Handle
 			return
 		}
 		logger = logger.With("name", r.URL.Path)
+		// TODO(lucas): defaultDays is 1.
```

Comment:
Intending to do this?

Reply:
Yes. I'll remove the comment and open a separate PR for this.

Reply:
See https://github.com/micromdm/nanodep/pull/6.